src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/DeviceTest.cs

```diff
@@ -204,8 +218,9 @@ protected static XDocument GetUI ()
 			}
 		}
 
-		protected static (int x, int y, int w, int h) GetControlBounds (string packageName, string uiElement, string text)
+		protected static (int x, int y, int w, int h) GetControlBounds (string packageName, string uiElement, string text, out bool found)
```

Comment:
not sure if crazy or sane, but should we just make the return value a nullable `(int x, int y, int w, int h)?`?